lib/1.4/dml-builtins.dml

```diff
@@ -908,20 +909,19 @@ method _get_attribute_attr(const _dml_attr_getset_info_t *_conf_info,
          --i) {
         local uint32 new_frag_size = id_info.dimsizes[i];
         local size_t new_no_items = no_items / new_frag_size;
-        local attr_value_t *new_items = new attr_value_t[new_no_items];
         for (local int j = 0; j < new_no_items; ++j) {
-            new_items[j] = SIM_alloc_attr_list(new_frag_size);
-            local attr_value_t *new_frag = SIM_attr_list(new_items[j]);
+            items_buf[j] = SIM_alloc_attr_list(new_frag_size);
+            local attr_value_t *new_frag = SIM_attr_list(items_buf[j]);
             for (local int k = 0; k < new_frag_size; ++k) {
                 new_frag[k] = items[j * new_frag_size + k];
             }
         }
-        delete items;
-        items = new_items;
+        (items, items_buf) = (items_buf, items);
```

Comment:
Oh, this was just a stray optimization I realized one can do, not related to immediate after.